### DISCUSSION TITLE
Newbie: Which board choice?

### DISCUSSION THREAD

**Discussion by nradina** [pschatzmann/arduino-audiokit#51] [Q&A] [answered]
Hi everyone,

I really love this resource and all the work you have done Phil! Thank you!

I picked up one of the Ai Thinker Audio Kit boards to give these projects a try.
Totally newbie question -- Which board do I select in the boards manager when using the Arduino IDE?

Thank you!

**Comment by pschatzmann** [comment, marked as answer]
Please read the FAQ of the AudioKit project

**Reply by nradina** [reply]
Thank you, Phil!
